lightning/src/ln/functional_tests.rs

```diff
@@ -2346,48 +2346,39 @@ fn claim_htlc_outputs_single_tx() {
 		}
 
 		let node_txn = nodes[1].tx_broadcaster.txn_broadcasted.lock().unwrap();
-		assert_eq!(node_txn.len(), 21);
+		assert_eq!(node_txn.len(), 9);
```

Comment:
As of commit b8ad51f, for test claim_htlc_outputs_single_tx comment lists: 
* 3 justice txn from ChannelMonitor
* 2 local txn from ChannelManager
* 2 bumped justice tx
* 2 local txn from ChannelMonitor

Reply:
I reorder number to make it clear, but 3 + 2 + 2 + 2 = 9? Txns aren't duplicate because we have a broadcast (local commitment+HTLC-timeout) from ChannelMonitor and ChannelManager, or did I miss something?